src/components/notification/notification_event.tsx

```diff
@@ -46,7 +46,7 @@ export type EuiNotificationHeadingLevel =
 
 export type EuiNotificationEventProps = Omit<
   EuiNotificationEventMetaProps,
-  'isRead' | 'onOpenContextMenu' | 'onRead'
+  'onOpenContextMenu' | 'onRead' | 'eventName'
```

Comment:
I also kept back in the `isRead` since they're the same prop type as from the meta prop.